src/main/webapp/js/calendarScript.js

```diff
@@ -13,22 +13,22 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-window.addEventListener('load', () => {
-  function initCalendar() {
-    new tui.Calendar('#calendar', {
-      defaultView: 'week',
-      useCreationPopup: true,
-      useDetailPopup: true,
-      disableDblClick: true,
-      disableClick: true,
-      isReadOnly: true,
-      scheduleView: ['time'],
-      taskView: false,
-      week: {
-        workweek: true,
-      },
-    });
-  }
+function initCalendar() {
```

Comment:
this file hasn't been converted to modules yet, so this still needs to be inside of the addEventListener.

Reply:
Wouldn't it also be a module because in the html we give the script type="module"?